packages/@aws-cdk/runtime-values/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "@aws-cdk/runtime-values",
   "version": "0.33.0",
   "description": "Runtime values support for the AWS CDK",
+  "deprecated": "This package has been deprecated",
```

Comment:
I would prefer a message that is more explicit about what the user should do... is there a replacement? Is the feature no longer useful? Is it harmful?